docs/sdk/tutorials/export_a_kili_project.md

```diff
@@ -1,22 +1,28 @@
 # How to export a Kili project
 ## Outline
-This tutorial explains the multiple ways to export a Kili project. It describes a per-label method involving label filtering and conversion, and also the solutions performing a full-project export. The methods are illustrated with code snippets.
+This tutorial explains the multiple ways to export a Kili project. It describes:
+
+ * A per-label method involving label filtering and conversion
+ * The solutions for performing a full-project export
+
+The methods are illustrated with code snippets.
+
 
 ## Export methods
 With Kili, once you have annotated enough assets, you can export the data programmatically to train a machine learning algorithm with it. There are several ways to do it:
 
- * Fetch the assets and/or the labels one by one using [`.assets`](https://python-sdk-docs.kili-technology.com/latest/sdk/asset/#kili.queries.asset.__init__.QueriesAsset.assets) or [`.labels`](https://python-sdk-docs.kili-technology.com/latest/sdk/label/#kili.queries.label.__init__.QueriesLabel.labels), perform the data transformation yourself, then write the data to one or several output files.
+ * Fetch the assets and/or the labels one by one using [`.assets`](https://python-sdk-docs.kili-technology.com/latest/sdk/asset/#kili.queries.asset.__init__.QueriesAsset.assets) or [`.labels`](https://python-sdk-docs.kili-technology.com/latest/sdk/label/#kili.queries.label.__init__.QueriesLabel.labels), perform the data transformation yourself and then write the data to one or several output files.
  * Export the whole project as a dataset. To do that, use the [`.export_labels`](https://python-sdk-docs.kili-technology.com/latest/sdk/label/#kili.queries.label.__init__.QueriesLabel.export_labels) method that creates an archive containing the labels in your chosen format.
 
 
 ## Preliminary steps
  1. Fetch the project ID from the Kili UI (in Settings / Admin):
    ![Get Project ID from UI](../../assets/get_project_id.jpg)
- 2. Ensure that your Kili API key as been set as an environment variable:
+ 2. Ensure that your Kili API key has been set as an environment variable:
  ```bash
  export KILI_API_KEY=<YOUR_API_KEY>
  ```
- 3. Install Kili if it has not been done already.
+ 3. In Kili has not been installed yet, install Kili.
```

Comment:
In -> if?

Reply:
Good catch. Trying to make changes too fast. Sorry